src/main/java/frc/robot/subsystems/conveyor/ConveyorIOReal.java

```diff
@@ -0,0 +1,38 @@
+package frc.robot.subsystems.conveyor;
+
+import com.revrobotics.CANSparkLowLevel;
+import com.revrobotics.CANSparkMax;
+import edu.wpi.first.units.Angle;
+import edu.wpi.first.units.MutableMeasure;
+import edu.wpi.first.units.Units;
+import edu.wpi.first.units.Velocity;
+import frc.robot.Constants;
+
+public class ConveyorIOReal implements ConveyorIO {
+
+    private CANSparkMax roller = new CANSparkMax(1, CANSparkLowLevel.MotorType.kBrushless);
+
+    public ConveyorIOReal() {
+        roller.restoreFactoryDefaults();
+        roller.setIdleMode(CANSparkMax.IdleMode.kCoast);
+        roller.enableVoltageCompensation(Constants.NOMINAL_VOLTAGE);
+        roller.setInverted(true);
+        for (int i = 1; i <= 6; i++) {
+            roller.setPeriodicFramePeriod(CANSparkLowLevel.PeriodicFrame.fromId(i), 1000);
```

Comment:
I'm sorry, I mislead you. I don't think neos work at 1000 hz. Check how much it was last year.

Reply:
That was eitan btw, gaia was signed in on my account.